gen/rewriter.jl

```diff
@@ -11,6 +14,11 @@ headerfiles = [
     "ogr_srs_api.jl"
 ]
 
+custom_rename = Dict{ASCIIString, ASCIIString}(
+    "gdalgetdriverbyname" => "gdalgetdriverbyname",
+    "ogrgetdriverbyname" => "ogrgetdriverbyname"
+)
+
```

Comment:
Yes it appears to be doing nothing, but because of the if/else construct, it prevents automatic renaming and overwriting method definitions for these two. I'll add a comment that clarifies this.